Demo-toRGB.py

```diff
@@ -3,6 +3,10 @@
 from PIL import Image
 
 def main():
+    """
+    This function turns all Ball picture and BG pictures to RGB (24 bit per pixel).
+    If the files are already RGB, the functions does nothing.
+    """
     ball_files, bg_files = os.listdir("Demo-BallSet"), os.listdir("Demo-BGSet")
     for ball in (ball_files):
         rgb_ball_image = Image.open("Demo-BallSet" + os.sep + ball).convert('RGB')
```

Comment:
You should probably migrate this code to its own function and call the function.

Reply:
Nevermind, did not realize it was on its own file.